src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -449,7 +446,7 @@ public void GivenWorkloadInstallItTreatsPreviewsAsSeparateFeatureBands()
             };
 
             var nugetDownloader = new MockNuGetPackageDownloader(dotnetRoot);
-            var manifestUpdater = new MockWorkloadManifestUpdater(manifestUpdates, tempDirManifestPath);
```

Comment:
I've been quietly facepalming every time I see this change. Thank you for deleting it.

Reply:
It was just an artifact left over from the creation of `IWorkloadManifestInstaller`, which now has this value built into it. When seeing unused properties like this, you generally should look at the code history to see if it was, in fact, no longer used or is actually some kind of glaring bug that was missed.